src/mod_roster.erl

```diff
@@ -334,7 +334,16 @@ create_sub_el(Items, Version) ->
     Params :: #{jid := jid:jid()},
     Extra :: gen_hook:extra().
 get_user_roster(Acc, #{jid := #jid{luser = LUser, lserver = LServer}}, #{host_type := HostType}) ->
-    NewAcc = case mongoose_acc:get(roster, show_full_roster, false, Acc) of
+    NewAcc = get_user_roster(Acc, LUser, LServer, HostType),
+    {ok, NewAcc}.
+
+-spec get_user_roster(mongoose_acc:t(),
+                      jid:luser(),
+                      jid:lserver(),
+                      mongooseim:host_type()
+                    ) -> mongoose_acc:t().
```

Comment:
I wouldn't separate that last parenthesis, if breaking the line I'd do so after the `->` 🤔